README.md

```diff
@@ -54,9 +54,7 @@ ventures.
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-No live Demo yet.
-<!-- [Live Demo Link](https://google.com)-->
-
+<[Live Demo Link](https://https://allenkays.github.io/Portfolio-setup-and-mobile-first/)>
```

Comment:
- [ ] The link here does open the expected page due to the duplicate "https://" in the link added to your README. 
 
you have this - `https://https://allenkays.github.io/Portfolio-setup-and-mobile-first/` instead of  `https://allenkays.github.io/Portfolio-setup-and-mobile-first/`
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/84156714/4b5787cd-2527-45ef-b8f1-3af0a3988d8f)

Please fix this.